src/routes/challan/list.tsx

```
@@ -168,13 +178,21 @@ export const ChallanList = ({ sales }: { sales?: boolean }) => {
     });
 
   return (
-    <List canCreate>
+    <List
+      canCreate
+      headerButtons={[
+        <CreateButton />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)